pkg/constants/constants.go

```diff
@@ -275,6 +275,24 @@ const (
 
 	// AWSCredsMount is the location where the AWS credentials secret is mounted for uninstall pods.
 	AWSCredsMount = "/etc/aws-creds"
+
+	// InstallLogsUploadProviderEnvVar is used to specify which object store provider is being used.
+	InstallLogsUploadProviderEnvVar = "HIVE_INSTALL_LOGS_UPLOAD_PROVIDER"
+
+	// InstallLogsUploadProviderAWS is used to specify that AWS is the cloud provider to upload logs to.
+	InstallLogsUploadProviderAWS = "aws"
+
+	// InstallLogsAWSCredentialsSecretRefEnvVar is the environment variable specifying what secret to use for storing logs.
+	InstallLogsAWSCredentialsSecretRefEnvVar = "HIVE_INSTALL_LOGS_AWS_CREDENTIALS_SECRET"
```

Comment:
Does this need to be a cloud-specific env var? Could it be `HIVE_INSTALL_LOGS_CREDENTIALS_SECRET` and knowing that `HIVE_INSTALL_LOGS_UPLOAD_PROVIDER` is `aws` already tells what credentials format will be in the secret?